server/rooms/commands/WhiteboardUpdateArrayCommand.ts

```diff
@@ -0,0 +1,31 @@
+import { Command } from '@colyseus/command'
+import { Client } from 'colyseus'
+import { IOfficeState } from '../../../types/IOfficeState'
+
+type Payload = {
+  client: Client
+  whiteboardId: string
+}
+
+export class WhiteboardAddUserCommand extends Command<IOfficeState, Payload> {
+  execute(data: Payload) {
+    const { client, whiteboardId } = data
+    const whiteboard = this.room.state.whiteboards.get(whiteboardId)
+    const clientId = client.sessionId
+
+    if (!whiteboard || whiteboard.connectedUser.includes(clientId)) return
```

Comment:
can `connectedUser` be a `Set<string>` instead of `string[]`? That way we have O(1) operations and also don't need to worry about duplicates.

Reply:
Changed to Set!